### ISSUE TITLE
IOU resluts problem about train_s4GAN.py 

### ISSUE THREAD

**Issue by tsrobcvai** [sud0301/semisup-semseg#15]
Hello, thank you for your outstanding research work!  I encountered such a problem. When I ran train_s4GAN.py (labeled-ratio 0.125/PASCAL-VOC Dataset/number_steps 35000), the best IOU I got was about 0.65, which is quite different from the 0.698 in the paper . The following is my visualization, Can you give me some suggestions, thank you!
![%AQ5$STJ_E1YZFS~9X6DG J](https://user-images.githubusercontent.com/73679440/116239626-7633bd00-a795-11eb-9ea8-ac932b0a938f.png)
![7JZ7PHW(6 QQH0CG_)_EA15](https://user-images.githubusercontent.com/73679440/116239634-78961700-a795-11eb-9745-eceb10b6fba8.png)

**Comment by sud0301** [comment]
Hi, 
Just a reminder, the paper reports performance of 69.8 mIOU with COCO+ImageNet pre-training. We report a performance of 65.4 in the paper when the network is pretrained only on ImageNet. Can you please verify whether you are using only ImageNet or ImageNet+COCO for pretraining the network.

**Comment by tsrobcvai** [comment]
> Hi,
> Just a reminder, the paper reports performance of 69.8 mIOU with COCO+ImageNet pre-training. We report a performance of 65.4 in the paper when the network is pretrained only on ImageNet. Can you please verify whether you are using only ImageNet or ImageNet+COCO for pretraining the network.

Yes it is! I just used only ImageNet for pretraining the network what you provided here，I thought only one kind of pre-training was needed (I'm a rookie). Could you tell me where I can find it ImageNet+COCO pretrained  model？And I have one more question，from the VAl_loss of my training, there is a problem of overfitting. Does this mean that the model can get better results after careful Tuning parameters? thank you！
